locales/en.json

```diff
@@ -106,6 +106,14 @@
       "confirmInvalid": "the two pins do not match"
     }
   },
+  "pin_login":{
+    "pin": {
+      "pinInfo": "Put the fingerprint on the sensor or enter the PIN",
```

Comment:
I would wait until we have the fingerprint functionality before adding it to the message - besides, the fingerprint login will not be available on all devices